ui/app/AppLayouts/Wallet/panels/ManageTokensPanel.qml

```diff
@@ -0,0 +1,378 @@
+                        Layout.preferredHeight: 20
+                        icon: "justify"
+                        color: Theme.palette.baseColor1
+                    }
+
+                    StatusRoundedImage {
+                        radius: groupedCommunityTokenDelegate.bgRadius
+                        Layout.preferredWidth: 32
+                        Layout.preferredHeight: 32
+                        image.source: model.communityImage
+                        showLoadingIndicator: true
+                        image.fillMode: Image.PreserveAspectCrop
+                    }
+
+                    StatusBaseText {
+                        text: model.communityName// + "(%1 -> %2)".arg(index).arg(model.customSortOrderNo)
+                        elide: Text.ElideRight
+                        maximumLineCount: 1
+                        font.weight: Font.Medium
+                    }
+
+                    StatusBaseText {
+                        Layout.leftMargin: -parent.spacing/2
+                        text: "<b>·</b> %1".arg(qsTr("%n asset(s)", "", communityDelegateRoot.childCount))
+                        elide: Text.ElideRight
+                        color: Theme.palette.baseColor1
+                        maximumLineCount: 1
+                        visible: !d.communityGroupsExpanded
+                    }
+
+                    Item { Layout.fillWidth: true }
+
+                    ManageTokenMenuButton {
+                        currentIndex: visualIndex
+                        count: d.controller.communityTokenGroupsModel.count
+                        isGroup: true
+                        groupId: model.communityId
+                        onMoveRequested: (from, to) => d.controller.communityTokenGroupsModel.moveItem(from, to)
+                        onShowHideGroupRequested: (groupId, flag) => d.controller.showHideGroup(groupId, flag)
+                    }
+                }
+
+                StatusListView {
+                    Layout.fillWidth: true
+                    Layout.preferredHeight: contentHeight
+                    model: d.controller.communityTokensModel
+                    interactive: false
+                    visible: d.communityGroupsExpanded
+
+                    displaced: Transition {
+                        NumberAnimation { properties: "x,y"; easing.type: Easing.OutQuad }
+                    }
+
+                    delegate: LocalTokenDelegate {
+                        isGrouped: true
+                        count: communityDelegateRoot.childCount
+                        dragEnabled: count > 1
+                        keys: ["x-status-draggable-community-token-item-%1".arg(model.communityId)]
+                        bgColor: Theme.palette.indirectColor4
+                        topInset: 2 // tighter "spacing"
+                        bottomInset: 2
+                        visible: communityDelegateRoot.communityId === model.communityId
```

Comment:
Yup, I agree that one is actually a PITA; either we get this performance hit, or the DND inside groups will not work. :/ I'll ask Ben if it would be acceptable to allow only programatic move inside those groups (which could be technically achieved using a SFPM)

Reply:
Or... I'm open for ideas :)